docs/pages/concepts/logical/relationships.md

```diff
@@ -20,12 +20,69 @@ Meshery supports a variety of relationships between components. Relationships ar
 
 Here is a list of the different types of relationships that Meshery supports:
 
-1. Edge - Network
-2. Edge - Mount
-3. Edge - Permission
-4. Edge - Firewall
-5. Heirarchical - Inventory
-6. Heirarchical - Parent
+1. Edge
+   1. Network
+   1. Firewall
+   1. Binding
```

Comment:
@RipulHandoo @MUzairS15, I’ve updated the logical relationships page and in doing so see that we need clarity in terms of the kinds, subtypes of relationships in that subtypes are being overloaded. Consider Binding and Mount. It may be that we need kind, type, subtype properties.